app/src/controllers/chefs.ts

```diff
@@ -1,84 +1,55 @@
+import config from 'config';
+
 import { chefsService } from '../services';
 import { isTruthy } from '../components/utils';
 import { IdentityProvider } from '../components/constants';
 
 import type { NextFunction, Request, Response } from 'express';
 import type { JwtPayload } from 'jsonwebtoken';
+import type { ChefsFormConfig, ChefsFormConfigData } from '../types/ChefsFormConfig';
+import type { ChefsSubmissionDataSource } from '../types/ChefsSubmissionDataSource';
 
 const controller = {
-  exportSubmissions: async (req: Request, res: Response, next: NextFunction) => {
-    try {
-      const response = await chefsService.exportSubmissions(req.params.formId);
-      res.status(200).send(response);
-    } catch (e: unknown) {
-      next(e);
-    }
-  },
-
-  getFormSubmissions: async (req: Request, res: Response, next: NextFunction) => {
+  getSubmissions: async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const response = await chefsService.getFormSubmissions(req.params.formId);
-
-      // IDIR users should be able to see all submissions
-      const filterToUser = (req.currentUser?.tokenPayload as JwtPayload).identity_provider !== IdentityProvider.IDIR;
-
-      if (isTruthy(filterToUser)) {
-        res
-          .status(200)
-          .send(
-            response.filter(
-              (x: { createdBy: string }) =>
-                x.createdBy.toUpperCase().substring(0, x.createdBy.indexOf('@idir')) ===
-                (req.currentUser?.tokenPayload as JwtPayload).idir_username.toUpperCase()
-            )
+      const cfg = config.get('server.chefs.forms') as ChefsFormConfig;
+      let formData = new Array<ChefsSubmissionDataSource>();
+
+      await Promise.all(
+        Object.values<ChefsFormConfigData>(cfg).map(async (x: ChefsFormConfigData) => {
+          const data = await chefsService.getFormSubmissions(x.formId);
+          formData = formData.concat(data);
+        })
+      );
+
+      /*
+       * Filter Data source
+       * IDIR users should be able to see all submissions
+       * BCeID/Business should only see their own submissions
+       */
+      const filterData = (data: Array<ChefsSubmissionDataSource>) => {
+        const filterToUser = (req.currentUser?.tokenPayload as JwtPayload).identity_provider !== IdentityProvider.IDIR;
```

Comment:
Just as a note - in the event we use a non-SSO Keycloak IAM we might not necessarily have the `identity_provider` attribute available. This is one of those few situations where I'm thinking it may be worth doing a defensive null check for tokenPayload?